docs/OnnxReleases.md

```diff
@@ -43,15 +43,15 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
     1. Bump opset version for ai.onnx domain in `onnx/defs/operator_sets.h` and `onnx/defs/schema.h` for use by future operator additions and changes.
         * For example, this [demo PR](https://github.com/onnx/onnx/pull/6001).
 
-## Upload release candidate to TestPyPI without offline steps (starting with onnx version 1.19)
+## Upload release candidate to PyPI
 
 * Go to "Actions" -> select ["Create Releases"](https://github.com/onnx/onnx/actions/workflows/create_release.yml) -> Push the button "Run workflow" with the following config:
 
 <img width="1078" height="1584" alt="RunWorkflow" src="https://github.com/user-attachments/assets/59c89418-395e-4c52-b0c6-a75ed4a6333b" />
```

Comment:
@andife Should we split an option for rc and release? This way we can ensure when rc is selected we check the version number has rc to prevent accidental release of the official version (e.g. when version number was not correctly updated to contain rc)

Reply:
yes, why not. Maybe a checkbox or dropdown which is "rc" by default